source/_integrations/iammeter.markdown

```diff
@@ -16,6 +16,8 @@ ha_integration_type: integration
 
 [IAMMETER](https://www.iammeter.com/) provides both a bi-directional single-phase energy meter([WEM3080](https://www.iammeter.com/products/single-phase-meter)) and a bi-directional three-phase energy monitor ([WEM3080T](https://www.iammeter.com/products/three-phase-meter)). Both of them can be integrated into Home Assistant.
 
+Besides the core integration,  you can also integrate the IAMMETER`s Wi-Fi energy meter into the Home Assistant by Modbus/TCP, MQTT, and HACS.
+For more details ,please refer to [summary:IAMMETER@Home Assistant](https://www.iammeter.com/docs/summary-home-assistant)
```

Comment:
This is recommending third-party integrations, which Home Assistant doesn't support and or recommend. Therefore, adding it here may suggest it is a recommendation by Home Assistant as well.

Going to friendly decline this PR for that reason.